doc/subsystems/networking/gptp_release_notes.rst

```diff
@@ -0,0 +1,55 @@
+gPTP stack for Zephyr
```

Comment:
Add a line in ``networking.rst`` to add this file (``gptp_release_notes.rst``) into the toctree directive list